doc/query.md

```diff
@@ -119,6 +195,86 @@ query.put("age", gt12);
 
 See below for supported operators (Selections -> Conditions).
 
+#### Text search
+
+A text clause may be used (if a text index exists) as part of a query to perform full text search term matching, phrase matching, prefix wildcard matching, etc.  A text clause can stand on its own as a query or can be part of a compound query (see below).  Text search supports SQLite Enhanced Query Syntax, see [SQLite full text query][ftsQuery] for more details on syntax that is possible with text search.
```

Comment:
Removed all "etc" references in the doc and reworded as suggested.  Done in ee198020c5b7e28d8fcf044f68f77f22fa1db7e2.